pkg/migration/controllers/migration.go

```diff
@@ -183,19 +261,47 @@ func (m *MigrationController) Handle(ctx context.Context, event sdk.Event) error
 	return nil
 }
 
-func (m *MigrationController) migrateVolumes(migration *stork_api.Migration) error {
-	storageStatus, err := getClusterPairStorageStatus(migration.Spec.ClusterPair, migration.Namespace)
-	if err != nil {
-		return err
+func (m *MigrationController) namespaceMigrationAllowed(migration *stork_api.Migration) bool {
+	// Restrict migration to only the namespace that the object belongs
+	// except for the namespace designated by the admin
+	if migration.Namespace != m.migrationAdminNamespace {
+		for _, ns := range migration.Spec.Namespaces {
```

Comment:
What does it mean to take a list of namespaces if we going to allow only one and that too the one of Migration itself?

Reply:
If the migration object is created in an AdminNamespace it can migrate any other namespace.